spec/syntax/remark_formatted.exp

```diff
@@ -43,7 +43,7 @@ FUNCTION remark_function(remark_parameter : STRING) : BOOLEAN;
     ;
   --"remark_repeat" function repeat scope - function repeat
   END_REPEAT;
-  remark_variable := QUERY(remark_query <* remark_variable() | TRUE
+  remark_variable := QUERY(remark_query <* remark_variable | TRUE
```

Comment:
What's the original EXPRESS? Does the original contain the parentheses? In the specs we want to test against real world files, so the specs should also reflect what is used in original EXPRESS files?